src/Common/Process.cpp

```diff
@@ -14,6 +14,7 @@
  *    being the original software.
  * 3. This notice may not be removed or altered from any source distribution.
  */
+#if defined( _WIN32 )
```

Comment:
I'd prefer it if you would just ifdef the contents of the functions themselves, as opposed to the entire class. It's less to clean up later when we actually implement processes on Linux!

Reply:
At the moment the function signatures include Windows-exclusive stuff - should I just redefine the functions to NOP or whatever?

Reply:
The `Run` function you can entirely if-def, but the ResultOf and OutputOf functions you can just if-def the function bodies